### PR TITLE
Add StructurePrinterOptions.IndentSize

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/GraphQLParser/Visitors/StructurePrinter.cs [hunk 1]
```diff
@@ -99,4 +109,9 @@ public class StructurePrinterOptions
     /// Print <see cref="ASTNode.Location"/> into the output.
     /// </summary>
     public bool PrintLocations { get; init; }
+
+    /// <summary>
+    /// The size of the horizontal indentation in spaces.
+    /// </summary>
+    public int IndentSize { get; init; } = 2;
```
[the remark this answers is not in the document]
Comment (sungam3r) [reply src/GraphQLParser/Visitors/StructurePrinter.cs:116]: All properties in both `SDLPrinterOptions` and `StructurePrinterOptions` have `init` and no issues reported so far. I would not change modifier to `set` for just one new property.

[the remark this answers is not in the document]
Comment (sungam3r) [reply src/GraphQLParser/Visitors/StructurePrinter.cs:116]: Correction - not all. I made `public int IndentSize { get; set; } ` in #294 but not intentionally.

[the remark this answers is not in the document]
Comment (Shane32) [reply src/GraphQLParser/Visitors/StructurePrinter.cs:116]: There’s a saying: don’t add fuel to the fire. 

Let’s fix the issue now while we are cleaning up this stuff.

[the remark this answers is not in the document]
Comment (sungam3r) [reply src/GraphQLParser/Visitors/StructurePrinter.cs]: done

**Pull request merged**